src/components/ebay-carousel/test/test.browser.js

```diff
@@ -80,13 +85,14 @@ describe('given the carousel starts in the default state with items', () => {
         });
 
         it('then it applies a translation', () => {
-            const translation = mock.itemWidth + constants.margin;
-            expect(list.style.transform).to.equal(`translateX(-${translation}px)`);
+            const { offsetLeft } = list.children[1];
+            expect(list.style.transform).to.equal(`translate3d(-${offsetLeft}px, 0px, 0px)`);
```

Comment:
The `translate3d` string is repeated a lot, maybe consider a helper?
```js
expect(getTranslation(list.style.transform)).to.equal(offsetLeft);
```

Reply:
👍